cmake/modules/Findcppzmq.cmake

```diff
@@ -8,8 +8,25 @@ find_path(cppzmq_INCLUDE_DIRS "zmq.hpp"
         )
 mark_as_advanced(cppzmq_INCLUDE_DIRS)
 
+# check for required minimum version for RooFitZMQ
+if(cppzmq_INCLUDE_DIRS)
+    SET(SAVE_CMAKE_REQUIRED_INCLUDES "${CMAKE_REQUIRED_INCLUDES}")
+    SET(CMAKE_REQUIRED_INCLUDES ${cppzmq_INCLUDE_DIRS} ${ZeroMQ_INCLUDE_DIRS})
+    SET(SAVE_CMAKE_REQUIRED_LIBRARIES "${CMAKE_REQUIRED_LIBRARIES}")
+    SET(CMAKE_REQUIRED_LIBRARIES libzmq)
+    check_cxx_source_runs("#include <zmq.hpp> \n int main () {if (CPPZMQ_VERSION_MAJOR > 4 || (CPPZMQ_VERSION_MAJOR == 4 && CPPZMQ_VERSION_MINOR >= 8)) {return 0;} else {return 1;} }" cppzmq_VERSION_COMPATIBLE)
+    SET(CMAKE_REQUIRED_INCLUDES "${SAVE_CMAKE_REQUIRED_INCLUDES}")
+    SET(CMAKE_REQUIRED_LIBRARIES "${SAVE_CMAKE_REQUIRED_LIBRARIES}")
+
+    if (NOT QUIET AND NOT cppzmq_VERSION_COMPATIBLE)
+        message("-- Version of zmq.hpp at ${cppzmq_INCLUDE_DIRS} too old, need at least 4.8.0.")
+    endif()
```

Comment:
This sort of message is not needed when you detect the version. The call to `find_package_handle_standard_args` handles that for you.